test/test-tcp-connect-error-after-write.c

```diff
@@ -74,7 +69,11 @@ TEST_IMPL(tcp_connect_error_after_write) {
   ASSERT_OK(r);
 
   r = uv_write(&write_req, (uv_stream_t*)&conn, &buf, 1, write_cb);
+#ifdef _WIN32
+  ASSERT_EQ(r, UV_EPIPE);
+#else
   ASSERT_EQ(r, UV_EBADF);
+#endif
 
   r = uv_tcp_connect(&connect_req,
```

Comment:
I might have made a rebase mistake?
```
2: not ok 314 - tcp_connect_error_after_write
2: # exit code -1073740791
2: # Output from process `tcp_connect_error_after_write`:
2: # Assertion failed in D:\a\libuv\libuv\test\test-tcp-connect-error-after-write.c on line 85: `r == 0` (-4047 == 0)
```

Reply:
Huh, STATUS_STACK_BUFFER_OVERRUN? I'll take a look.

Reply:
We used to have an ASAN test until it seemed that GithubActions updated something and broke it: https://github.com/libuv/libuv/issues/4210

Reply:
seems that ASAN didn't have anything to report
```
not ok 314 - tcp_connect_error_after_write
# exit code -1073740791
# Output from process `tcp_connect_error_after_write`:
# Assertion failed in D:\a\libuv\libuv\test\test-tcp-connect-error-after-write.c on line 85: `r == 0` (-4047 == 0)
```